types/index.d.ts

```diff
@@ -0,0 +1,434 @@
+
+import L from "leaflet";
+
+type IndoorMapId = string;
+type IndoorMapFloorId = number;
+type IndoorMapFloorIndex = number;
+
+type ElevationMode = "heightAboveGround" | "heightAboveSeaLevel";
+
+type Vector3 = [number, number, number] | {
+    x: number;
+    y: number;
+    z: number;
+};
+
+type Vector4 = [number, number, number, number] | {
+    x: number;
+    y: number;
+    z: number;
+    w: number;
+};
+
+type Color = string | Vector3 | Vector4 | {
+    r: number;
+    g: number;
+    b: number;
+    a?: number;
+};
+
+/* Wrld.Map */
+
+type MapOptions = L.MapOptions & {
+    canvasId?: string;
+    width?: number;
+    height?: number;
+    indoorsEnabled?: boolean;
+    displayEntranceMarkers?: boolean;
+    coverageTreeManifest?: string;
+    environmentThemesManifest?: string;
+    headingDegrees?: number;
+    trafficEnabled?: boolean;
+    trafficDisabledWhenEnteringIndoorMaps?: boolean;
+    indoorLabelsAlwaysHidden?: boolean;
+    drawClearColor?: Color;
+    indoorMapBackgroundColor?: Color;
+    viewVerticallyLocked?: boolean;
+    targetVSyncInterval?: number;
+    frameRateThrottleWhenIdleEnabled?: boolean;
+    idleSecondsBeforeFrameRateThrottle?: number;
+    throttledTargetFrameIntervalMilliseconds?: number;
+};
+
+namespace Map {
+
+    // eslint-disable-next-line no-unused-vars
+    type SetViewOptions = {
+        headingDegrees: number;
+        tiltDegrees: number;
+        animate: boolean;
+        durationSeconds: number;
+        allowInterruption: boolean;
+    };
+}
+
+// eslint-disable-next-line no-redeclare
+interface Map extends L.Map {
+    getAltitudeAtLatLng(latLng: L.LatLngExpression): number;
```

Comment:
Is this correct.  I think getAltitudeAtLatLng can only accept L.LatLng or { lat: number; lng: number } but L.LatLngExpression also includes [number, number]